benchalerts/benchalerts/pipeline_steps/conbench.py

```diff
@@ -104,6 +104,7 @@ def _get_one_run_comparison(self, run_id: str) -> Optional[RunComparisonInfo]:
             raise
 
         run_comparison = RunComparisonInfo(
+            conbench_api_url=self.conbench_client._base_url,
```

Comment:
Maybe we should un-underscore this? Or we could get it from the env var, which is always set when this is run, I'm pretty sure, but that's less direct.

Reply:
Yeah I considered the env var but the client has some nice handling built in: 
https://github.com/conbench/conbench/blob/be2f5978c6af0291bab0accda1acfc65eb4a724d/benchclients/python/benchclients/conbench.py#L98

Someday we'll set up a nice integration test suite and won't have to worry about our libraries using each others' private attributes.